app.rb

```diff
@@ -33,25 +35,23 @@ def list_persons(display_num: false)
 
   def list_books(display_num: false)
     # if there are no books
-    return puts "\nERROR:\nThere are no books created yet!\n\n" if books.nil? || books.empty?
+    return puts "\nERROR:\nThere are no books created yet!\n\n" if @books.nil? || @books.empty?
```

Comment:
- Great job is done handling the case when one or more of the files are missing to avoid the app from throwing errors 🥇